src/brpc/redis.h

```diff
@@ -107,21 +107,21 @@ class RedisRequest : public ::google::protobuf::Message {
     bool SerializeTo(butil::IOBuf* buf) const;
 
     // Protobuf methods.
-    RedisRequest* New() const;
-    void CopyFrom(const ::google::protobuf::Message& from);
-    void MergeFrom(const ::google::protobuf::Message& from);
+    RedisRequest* New() const override;
```

Comment:
Why "override" is needed only in "redis.h" while not in "thrift_message.h" or "nshead_message.h" or other headers files?

Reply:
Look like the redis.h is included by some very common header, and my project includes the "redis.h" indirectly.

Reply:
I see "redis.h" is included in "server.h".
It is ok to add "override" to "redis.h".